doc/releases/changelog-dev.md

```diff
@@ -76,6 +76,9 @@
 
 <h3>Internal changes ⚙️</h3>
 
+* Add `PCPhase` native compiler and runtime operator support.
+  [(#1987)](https://github.com/PennyLaneAI/catalyst/pull/1987)
+
 * Add `SingleExcitation` and `DoubleExcitation` to native `RUNTIME_OPERATIONS` and `RuntimeCAPI`.
   [(#1980)](https://github.com/PennyLaneAI/catalyst/pull/1980)
```

Comment:
Aren't these user-facing changes? They can now use these gates without decomposition in their programs. We should also mention that this significantly speeds up simulation runtime in the changelog (the "so what").

Reply:
yeah, I was thinking the same. I'm benchmarking QSVT w/ this PR for the presentation. I'll update the changelog adding more details about the "so what" :smile:

Reply:
Update [10a0616](https://github.com/PennyLaneAI/catalyst/pull/1987/commits/10a06168edab2f1425e3f952823cc31c74806327)

Reply:
benchmark script is still running. I'll share the results later